test/intl402/DateTimeFormat/timezone-canonicalized.js

```diff
@@ -1,35 +0,0 @@
-// Copyright 2012 Mozilla Corporation. All rights reserved.
-// This code is governed by the BSD license found in the LICENSE file.
-
-/*---
-es5id: 6.4_c
-description: >
-    Tests that additional time zone names, if accepted, are handled
-    correctly.
-author: Norbert Lindenberg
----*/
-
-// canonicalization specified in conformance clause
-var additionalTimeZoneNames = {
-    "Etc/GMT": "UTC",
-    "Greenwich": "UTC",
-    "PRC": "Asia/Shanghai",
-    "AmErIcA/LoS_aNgElEs": "America/Los_Angeles",
-    "etc/gmt+7": "Etc/GMT+7"
```

Comment:
Aren't these still valid? Or are they rolled into a different test now?

Reply:
This API no longer canonicalizes identifiers, so the first three tests don't apply anymore. The latter two are covered by the lowercase/uppercase test. Well, I guess "AmErIcA/LoS_aNgElEs" isn't if you're splitting hairs, but I didn't really think that a combo of upper and lower case was important to test if we're already testing all lower and all upper case for each ID. Holler and un-resolve if you disagree!